persistence/src/commonMain/db_user/com/wire/kalium/persistence/MessageAttachmentDraft.sq

```diff
@@ -0,0 +1,65 @@
+import com.wire.kalium.persistence.dao.QualifiedIDEntity;
+import kotlin.Boolean;
+import kotlin.Float;
+import kotlin.Int;
+import kotlin.String;
+
+CREATE TABLE MessageAttachmentDraft (
+      attachment_id TEXT NOT NULL,
+      version_id TEXT NOT NULL,
+      conversation_id TEXT AS QualifiedIDEntity NOT NULL,
+      mime_type TEXT NOT NULL,
+      file_name TEXT NOT NULL,
+      file_size INTEGER NOT NULL,
+      data_path TEXT NOT NULL,
+      node_path TEXT NOT NULL,
+      upload_status TEXT NOT NULL,
+      asset_width INTEGER AS Int,
+      asset_height INTEGER AS Int,
+      asset_duration_ms INTEGER,
+
+    FOREIGN KEY (conversation_id) REFERENCES Conversation(qualified_id) ON DELETE CASCADE ON UPDATE CASCADE,
+    PRIMARY KEY (attachment_id)
+);
+
+getDraft:
+SELECT * FROM MessageAttachmentDraft WHERE attachment_id = ?;
+
+getDrafts:
+SELECT * FROM MessageAttachmentDraft WHERE conversation_id = ?;
```

Comment:
for this you need to create an INDEX with the conversationID, since by default FOREIGN KEY is not indexed in sql

```SQLite
CREATE INDEX attachment_draft_conversation_id_index ON MessageAttachmentDraft(conversation_id);